src/schema/helpers/fn-schema.ts

```diff
@@ -0,0 +1,132 @@
+import {
+  booleanTypeFactory,
+  CoreType,
+  dateTypeFactory,
+  numberTypeFactory,
+  stringTypeFactory,
+  arrayTypeFactory,
+  objectTypeFactory,
+} from '../types';
+import { is, isTypeOf } from '../../utils/is-type';
+import { BuildSchemaError } from '../errors';
+import { Schema, SchemaDef, ModelObject } from '../schema';
+
+const getType = (v) => ![, null].includes(v) && (v.name || v);
+
+/**
+ * Parse definition to get a schema instance
+ * @function
+ * @public
+ *
+ * @param {Schema|Object} obj
+ * @returns {Schema}
+ * @throws {Error}
+ *
+ * @example
+ *  ```ts
+ *    const schema = createSchema({name: String, hasChild: {type: Boolean, default: true}});
+ *  ```
+ */
+export const createSchema = (obj: Schema | SchemaDef): Schema => {
+  if (obj instanceof Schema) {
+    return obj;
+  }
+  const fields: CoreType[] = [];
+  const keys = Object.keys(obj);
+  for (const _key of keys) {
+    const opts = _parseType(obj[_key]);
+    if (!opts.type) {
+      throw new BuildSchemaError(`Property ${_key} required type`);
```

Comment:
Should be: `Property ${_key} is a required type`

Reply:
Fixed: Property ${_key} is a required type